.gitignore

```diff
@@ -5,10 +5,15 @@
 /tmp/
 
 # dependencies
-/bower_components/
 /node_modules/
 
+# storybook
+/storybook-static/
+/.storybook/preview-head.html
```

Comment:
Added these to the gitignore as `ember test` now generates `/.storybook/preview-head.html`